### ISSUE TITLE
Alert selector not opening

### ISSUE THREAD

**Issue by natorion** [catapult-project/catapult#2401]
Chrome:  53.0.2763.0
OS: Mac

Repro:

- Goto https://chromeperf.appspot.com/alerts?sheriff=ARC%20Perf%20Sheriff
- Click on alert selector on the top left
- No drop box is showing

This still works in Chrome Stable (51). No clue if this is a Chrome or Catapult problem.

**Comment by anniesullie** [comment]
@Apeliotes does this happen on the polymer 1.0 branch?

This looks like a polymer 0.5 bug.

On Canary, I see this JS error:
```
web-animations-next-lite.min.js:15 Uncaught TypeError: Failed to execute 'animate' on 'Element': 'function (e){function f(a,b,c){return 3*a*(1-c)*(1-c)*c+3*b*(1-c)*c*c+c*c*c}for(var g=0,h=1;;){var i=(g+h)/2,j=f(a,c,i);if(Math.abs(e-j)<.001)return f(b,d,i);e>j?g=i:h=i}}' is not a valid value for easing
b.newUnderlyingPlayerForAnimation @ web-animations-next-lite.min.js:15
_rebuildUnderlyingPlayer @ web-animations-next-lite.min.js:15
b.Player @ web-animations-next-lite.min.js:15
play @ web-animations-next-lite.min.js:15
b.newUnderlyingPlayerForGroup @ web-animations-next-lite.min.js:16
_rebuildUnderlyingPlayer @ web-animations-next-lite.min.js:15
b.Player @ web-animations-next-lite.min.js:15
play @ web-animations-next-lite.min.js:15
transitionOpened @ paper-dropdown-transition.html:152
go @ core-transition-css.html:174
renderOpened @ core-overlay.html:407
(anonymous function) @ core-overlay.html:364
(anonymous function) @ polymer.js:8988
```

**Comment by natorion** [comment]
Chrome Stable just got updated to Chrome 52. Selector stopped working. Currently Chromeperf is not usable as a sheriffing tool.

**Comment by anniesullie** [comment]
I'm really sorry about this. This will be fixed by polymer 1.0 migration but we've hit some blocking bugs which we are actively working on: https://github.com/catapult-project/catapult/issues?q=is%3Aissue+is%3Aopen+label%3A%22Polymer+1.0+Migration%22+label%3A%22Perf+Dashboard%22

**Comment by anniesullie** [comment]
This has been fixed in the polymer 1.0 version of the perf dashboard which was launched last week.
